great_expectations/expectations/core/expect_column_max_to_be_between.py

```diff
@@ -150,13 +150,7 @@ class ExpectColumnMaxToBeBetween(ColumnExpectation):
                 "variables": {
                     "strict_min": False,
                     "strict_max": False,
-                    "false_positive_rate": 0.05,
-                    "estimator": "bootstrap",
-                    "n_resamples": 9999,
-                    "random_seed": None,
-                    "quantile_statistic_interpolation_method": "nearest",
-                    "quantile_bias_correction": False,
-                    "quantile_bias_std_error_ratio_threshold": None,
+                    "estimator": "exact",
```

Comment:
Could this be an enum?

Reply:
I would prefer to be able to use `ExactNumericRangeEstimator` here, but we would need to think about how to add that to the schema in quite a few different places and we also apparently have limitations with `instantiate_class_from_config` in doing so.